### ISSUE TITLE
JSDoc@enum Indicates whether jsdoc@enum is supported?

### ISSUE THREAD

**Issue by Github-Jackson** [thqby/vscode-autohotkey2-lsp#380]
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/53117644/cb51f79d-1401-4bb6-bb1b-e4c3d16ec9c1)
Is there any way to enumerate the parameter values of the function like the built-in function shown in the above figure?
I tried @enum,@typedef and they didn't work
```ahk2
/**
 * @param {'one'|'twe'} Enum 
 */
TestFn(Enum){
}
```
---
This extension is really great, DLLCall can parse DLL exported functions!
Is there any way to enable my encapsulated function (NativeCall) to have similar code suggestions?
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/53117644/09d21932-0aad-40fd-b738-46576b88e745)

**Comment by thqby** [comment]
> Is there any way to enable my encapsulated function (NativeCall) to have similar code suggestions?
No

**Comment by Github-Jackson** [comment]
Thank you! enum is already supported, but doesn't seem to work in Parameter Hints.
`Parameter Hints` : The Parameter suggestion mode that vscode triggers when calling a method, and the `ctrl+shift+space` Trigger Parameter Hints.
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/53117644/23be92f2-749f-49cb-830d-568b1ba2797a)


Exit Parameter suggestion mode works fine!
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/53117644/abb32f56-4bca-4d62-8f4d-64c0ceb50036)

**Comment by Github-Jackson** [comment]
enum in #include script does not seem to work.
```js
;-----EnumDoc.ahk------
/** @param {'one'|'twe'} Flags  */
TestEnum(Flags){
}
;-----main.ahk-----
#Include EnumDoc.ahk
TestEnum('') /** lost enum hint*/
```
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/53117644/81c20cea-57bb-4aef-bb41-c4c5006699a1)

**Comment by thqby** [comment]
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/28240311/256f3bb1-f8ba-445e-9f04-d85f988d124b)

**Comment by Github-Jackson** [comment]
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/53117644/e29cbb2c-a7a8-4f9c-a0d1-ddfad89b06f2)


In my current extension(v2.1.5) environment, I can provide suggest normally after restarting vscode. Then typing a new code, enum, won't work.
Maybe the `Fixed #380` https://github.com/thqby/vscode-autohotkey2-lsp/commit/7934025d9d7821cc5dd775fac75aaeac4fbea4b1 update won't have this problem.
